src/warnet/admin.py

```diff
@@ -1,4 +1,5 @@
 import os
+import subprocess
```

Comment:
do we need subprocess? iirc, we have a warnet command runner (run_command, or stream_command?)